src/beam_stats_consumer_statsd.erl

```diff
@@ -143,16 +143,16 @@ try_to_connect_if_no_socket(#state{sock=none, src_port=SrcPort}=State) ->
 ) ->
     [binary()].
```

Comment:
And if `iolist_to_binary` is removed, this `[binary()]` should now be `[iolist()]`.